src/App.js

```diff
@@ -50,9 +52,20 @@ export default function App() {
       });
   }, []);
 
+  function search(participantData) {
+    return participantData.filter(participant => {
+      const address = participant.properties.address.toLowerCase();
+      const name = participant.properties.business_name?.toLowerCase();
+
+      return address.includes(query) || (name && name.includes(query));
+    });
+  }
+  console.log(query);
```

Comment:
Please delete this console log. In your final PR there should be no console logs.

Reply:
Ah yes! i will delete this console log..
But there is something strange around this filter function.. if i type capital letters in my input field there are no results. Lower case letters are working..

Reply:
Is fixed